labs/coding-101-rest-basics-ga/7.md

```diff
@@ -9,7 +9,7 @@
 		* `A.` Select **GET** in the Method drop down
 	* **URL**
 		* `B.` Enter **http://{APIC-EMController}/api/v1/host**
-            * If you are not using your APIC-EM Controller, use the APIC-EM Lab: https[]()://devnetapi.cisco.com/sandbox/apic_em.
+            * If you are not using your APIC-EM Controller, use the APIC-EM Lab: https[]()://sandboxapicem.cisco.com.
```

Comment:
Same odd markup here, is it supposed to be that way?

Reply:
I tested, and I'm not sure why they did it that way originally. It works without it.